package.json

```diff
@@ -22,7 +22,8 @@
     "start:live": "ember serve --proxy https://crates.io",
     "start:local": "ember serve --proxy http://127.0.0.1:8888",
     "start:staging": "ember serve --proxy https://staging-crates-io.herokuapp.com",
-    "test": "ember exam --split=2 --parallel"
+    "test": "ember exam --split=2 --parallel",
+    "test-coverage": "COVERAGE=true npm run test && ember coverage-merge && rm -rf coverage_* coverage/coverage-summary.json && nyc report"
```

Comment:
why do we `rm -rf` here?

Reply:
ember-cli-code-coverage, when called with `--parallel` option, outputs to multiple `coverage_[hash]` directories. These pile up quickly, since there new ones are created after each test run. Hence `coverage_*` removal.

`coverage-summary.json` is removed to prevent nyc issue (see: https://github.com/kategengler/ember-cli-code-coverage/pull/173#issuecomment-381274578)